qubesmanager/qube_manager.py

```diff
@@ -238,6 +243,56 @@ def check_if_vm_has_shutdown(self):
             self.restart_vm_if_needed()
 
 
+# pylint: disable=too-few-public-methods
+class StartVMThread(QtCore.QThread):
+    def __init__(self, vm):
+        QtCore.QThread.__init__(self)
+        self.vm = vm
+        self.msg = None
+
+    def run(self):
+        try:
+            self.vm.start()
+        except exc.QubesException as ex:
+            self.msg = ("Error starting Qube!", str(ex))
+
+
+# pylint: disable=too-few-public-methods
+class UpdateVMThread(QtCore.QThread):
+    def __init__(self, vm):
+        QtCore.QThread.__init__(self)
+        self.vm = vm
+        self.msg = None
+
+    def run(self):
+        try:
+            if self.vm.qid == 0:
+                subprocess.check_call(
+                    ["/usr/bin/qubes-dom0-update", "--clean", "--gui"])
+            else:
+                if not self.vm.is_running():
+                    self.vm.start()
+                self.vm.run_service("qubes.InstallUpdatesGUI",\
+                        user="root", wait=False)
+        except (ChildProcessError, exc.QubesException) as ex:
+            self.msg = ("Error on Qube update!", str(ex))
```

Comment:
I think in most other places 'qube' is not capitalized, I would not capitalize it here.

Reply:
There are other places with capitalized Qube, I will change them too.

Reply:
Oh, well maybe better I can wait for determine exactly when use one or other. e.g.
https://github.com/QubesOS/qubes-manager/blob/77cf9bf1d23301a54f2849c0c5d8ce980b14fc0a/qubesmanager/qube_manager.py#L690